systems/controllers/osc/osc_tracking_data.h

```diff
@@ -0,0 +1,428 @@
+#pragma once
+
+#include <Eigen/Dense>
+#include <string>
+#include <vector>
+
+#include "drake/multibody/rigid_body_tree.h"
+#include "drake/common/trajectories/piecewise_polynomial.h"
+#include "drake/common/trajectories/exponential_plus_piecewise_polynomial.h"
+
+#include "systems/framework/output_vector.h"
+
+#include "systems/controllers/osc/osc_user_defined_traj.h"
+
+namespace dairlib {
+namespace systems {
+namespace controllers {
+
+/// OscTrackingData is a virtual class
+
+/// Input of the constructor:
+/// - dimension of the output/trajectory
+/// - gains of PD controller
+/// - cost weight
+/// - a flag indicating the trajectory is a constant
+/// - a flag indicating the trajecotry has exponential term (that is, the traj
```

Comment:
Correct

Reply:
I felt unsatisfied when I had to add an extra flag just because I couldn't use `AbstractValue<Trajectory> `, but it seems it's possible to initialize the abstract value now.